mappers/bluetooth_mapper/configuration/config.yaml

```diff
@@ -0,0 +1,42 @@
+mqtt:
+  mode: 0       # 0 -internal mqtt broker  1 - external mqtt broker
+  server: tcp://127.0.0.1:1883 # external mqtt broker url.
+  internal-server: tcp://127.0.0.1:1884 # internal mqtt broker url.
+device-name: CC2650 SensorTag
+action-manager:
+  actions:
+    - name: IRTemperatureConfiguration
+      enable: true
```

Comment:
enable -> performImmediately ?

Reply:
have renamed it to perform-iImmediately. Thanks

Reply:
sure.